### PR TITLE
Add K5 schedule and grades e2e tests

### PR DESCRIPTION
**Pull request by davidvarga93**
refs: MBL-15589
affects: Student
release note: none

test plan:

### CODE REVIEW

**Comment by inst-danger** [comment]
<details><summary>Student.ipa</summary>
  <a href="https://app.bitrise.io/artifact/90144165/p/b09e442064621aff8654d7f6e9f922f5">
    
  </a>
</details>
<details><summary>Teacher.ipa</summary>
  <a href="https://app.bitrise.io/artifact/90144199/p/08799da563b3fd011faea4900fff9605">
    
  </a>
</details>
<details><summary>Parent.ipa</summary>
  <a href="https://app.bitrise.io/artifact/90144076/p/cfc338e1d38b61cac091a72bb31a8aae">
    
  </a>
</details>

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 91.86% | 91.86% | 0%

[MBL-15589](https://instructure.atlassian.net/browse/MBL-15589)

#### Affected Apps: Student
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: TestsFoundation/TestsFoundation/Element/K5NavigationBar.swift [hunk 1]
```diff
@@ -19,9 +19,17 @@
...
     public static var homeroom: Element {
         return app.find(label: "Homeroom")
     }
+
+    public static var schedule: Element {
+        return app.find(label: "Schedule")
+    }
+
+    public static var grades: Element {
+        return app.find(label: "Grades")
```
Comment (vargaat) [inline TestsFoundation/TestsFoundation/Element/K5NavigationBar.swift:33]: In these cases the `return` statement can be omitted. Sometimes we even make the `var` a one-liner if it's short.

File: TestsFoundation/TestsFoundation/CoreUIElements/K5/K5Grades.swift [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+public enum K5CourseGrades: ElementWrapper {
+
+    public static var emptyGradesForCourse: Element {
+        return app.find(label: "No Assignments")
+    }
+
+    public static func gradedPointsOutOf(actual: String, outOf: String) -> Element {
+    return app.find(label: "Grade, \(actual) out of \(outOf)")
```
Comment (vargaat) [inline TestsFoundation/TestsFoundation/CoreUIElements/K5/K5Grades.swift:35]: Indentation is off. Return could be omitted if you feel so.

**Review by vargaat — COMMENTED** [review]
Did this pass on CI?

**Comment by davidvarga93** [comment]
Nightly: https://app.bitrise.io/build/75f2ac0a-dab5-42d6-a4f4-8f1fe70bbaf8#?tab=log

**Review by balintbartok — APPROVED** [review]

**Review by vargaat — APPROVED** [review]

**Pull request merged**